Python/traceback.c

```diff
@@ -415,6 +415,9 @@ _Py_FindSourceFile(PyObject *filename, char* namebuf, size_t namelen, PyObject *
     npath = PyList_Size(syspath);
 
     open = PyObject_GetAttr(io, &_Py_ID(open));
+    if (open == NULL) {
+        goto error;
```

Comment:
@ZeroIntensity, should we link the issue here? It may seem odd in the wild.

```suggestion
        // See GH-142737.
        goto error;
```

Reply:
Nah, I think it's fine to keep it as-is.